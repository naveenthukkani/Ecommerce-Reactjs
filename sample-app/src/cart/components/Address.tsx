import React from 'react';
import "../Style.css";

export const Address = () => {
return (
    <div className="Address-sub-conatiner">
        <p className="Heading">Shipping Address:</p>
        <textarea className="Address-box">
        </textarea>
    <div>
      <button className="Address-buttons">Save Address</button>
      <button className="Address-buttons">Edit Address</button>
    </div>    
    </div>
)
};