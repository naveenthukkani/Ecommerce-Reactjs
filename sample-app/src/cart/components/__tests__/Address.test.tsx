import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {Address} from '../Address';


describe("Address", () =>{

    it('renders address component correctly', () => {
        const component = shallow(<Address />);
        expect(toJson(component)).toMatchSnapshot();
    });

});