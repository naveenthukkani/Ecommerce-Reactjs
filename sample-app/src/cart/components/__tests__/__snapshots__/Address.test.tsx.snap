// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address renders address component correctly 1`] = `
<div
  className="Address-sub-conatiner"
>
  <p
    className="Heading"
  >
    Shipping Address:
  </p>
  <textarea
    className="Address-box"
  />
  <div>
    <button
      className="Address-buttons"
    >
      Save Address
    </button>
    <button
      className="Address-buttons"
    >
      Edit Address
    </button>
  </div>
</div>
`;
